website/config/routes.js

```diff
@@ -783,6 +783,7 @@ module.exports.routes = {
   // =============================================================================================================
   // Redirects for external links from the Fleet UI & CLI, including to fleetdm.com and to external websites not
   // maintained by Fleet. These help avoid broken links by reducing surface area of links to maintain in the UI.
+  'GET /learn-more-about/reinstall-software': 'https://fleetdm.com/guides/automatic-software-install-in-fleet#templates-for-policy-queries',
```

Comment:
Can you change the target of this redirect to be a relative link to the guide?

```suggestion
  'GET /learn-more-about/reinstall-software': '/guides/automatic-software-install-in-fleet#templates-for-policy-queries',
```